src/Collection/Collection.router.ts

```diff
@@ -244,67 +247,63 @@ export class CollectionRouter extends Router {
   ): Promise<{ collection: FullCollection; items: FullItem[] }> => {
```

Comment:
This method did not have a `withCollectionAuthorization` but I'm not sure why or if it's necessary